lab-yunjoo/app/component/list/index.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('./list.scss');
+const angular = require('angular');
+angular.module('demoApp').directive('appList', function(){
+  return {
+    restrict: 'E',
+    replace: true,
+    template: require('./list.html'),
+    controller: ['$log','listService','noteService',ListController],
```

Comment:
This works fine, but if you ever try to test the controller, you won't be able to--you'll only be able to test the directive. It would be better if your defined the controller on the module and then just specified it with a string here. That also makes it more readable.